packages/realm-tools/src/realm-schema.ts

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import Realm from "realm";
+import { parse } from "ts-command-line-args";
+import * as fs from "fs";
+
+type Relationship = {
+  from: string;
+  to: string;
+};
+
+type ExtractSchemaArgs = {
+  inputFileName?: string;
+  outputFileName?: string;
+  format?: string;
+  help?: boolean;
+};
+
+type FormatterFunction = (r: Realm) => void;
+type Formatters = {
+  [key: string]: FormatterFunction;
+};
+
+const mermaid: FormatterFunction = (realm: Realm) => {
+  writer("classDiagram");
```

Comment:
[NIT] The output markdown file does not contain the `mermaid` wrapper syntax.  I think it should.